tests/test_app.py

```diff
@@ -104,133 +121,89 @@ def test_get_property_invalid_path_404(self, mock_response, mock_get):
         mock_response.status_code = 404
         mock_response.raise_for_status.side_effect = requests.exceptions.HTTPError
         mock_get.return_value = mock_response
-        street_address_url = "Trewithy Court"
-        response = self.app.get('/properties/%s' % street_address_url)
+        street_address_url = 'Trewithy Court'
+        response = self.app.get('/properties/{}'.format(street_address_url))
         assert response.status_code == 404
 
-    @mock.patch('service.server.get_property_address', return_value=one_DB_result)
+    @mock.patch('service.server.get_property_address', return_value=single_elastic_search_result.hits)
     @mock.patch('requests.post')
     def test_get_property_address_called_correctly(self, mock_post, mock_get_property_address):
-        search_query = "PL6%208RU/PATTINSON%20DRIVE_100"
-
-        query_dict = {
-            'postcode': 'PL6 8RU',
-            'street': 'PATTINSON DRIVE',
-            'paon': '100',
-        }
+        postcode = 'PL6_8RU'
+        joined_address_fields = '100_PATTINSON_DRIVE_PL6_8RU'
```

Comment:
a bit surprised to see PL6_8RU in the `joined_address_fields` here, am I missing something?